macs/guest/apply.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+echo "apply started at $(date)" | nc -w0 -u 10.172.170.1 1514
+
+printf '\n*.*\t@10.172.170.1:1514\n' | tee -a /etc/syslog.conf
+pkill syslog
+pkill asl
+
+exec 3>&1
+exec 2> >(nc -u 10.172.170.1 1514)
+exec 1>&2
+
+PS4='${BASH_SOURCE}::${FUNCNAME[0]}::$LINENO '
+set -o pipefail
+set -ex
+date
+
+function finish {
+    set +e
+    cd /
+    sleep 1
+    umount -f /Volumes/CONFIG
+}
+trap finish EXIT
+
+cat <<EOF | tee -a /etc/ssh/sshd_config
+PermitRootLogin prohibit-password
+PasswordAuthentication no
+PermitEmptyPasswords no
+ChallengeResponseAuthentication no
+EOF
+
+launchctl stop com.openssh.sshd
+launchctl start com.openssh.sshd
+
+
+cd /Volumes/CONFIG
+
+cp -r ./etc/ssh/ssh_host_* /etc/ssh
+chown root:wheel /etc/ssh/ssh_host_*
+chmod 600 /etc/ssh/ssh_host_*
+cd /
+
+echo "%admin ALL = NOPASSWD: ALL" | tee /etc/sudoers.d/passwordless
+
+(
+    # Make this thing work as root
+    export USER=root
+    export HOME=~root
+    export ALLOW_PREEXISTING_INSTALLATION=1
+    env
+    curl https://nixos.org/releases/nix/nix-2.1.3/install > ~nixos/install-nix
```

Comment:
I feel like the nix version should be configurable somewhere central so we can update all of the fleet easily